src/components/concerts/ConcertCard/ConcertCard.js

```diff
@@ -24,37 +24,41 @@ const ITEM_HEIGHT = 48;
 const ConcertCard = ({ concert }) => {
   const navigate = useNavigate();
 
+  const { title, venue, time, coverImage, artists } = concert;
+
   const [anchorEl, setAnchorEl] = useState(null);
   const open = Boolean(anchorEl);
+
   const handleClick = (e) => {
     setAnchorEl(e.currentTarget);
   };
+
   const handleClose = () => {
     setAnchorEl(null);
   };
 
   return (
     <Card
       sx={{ maxWidth: 345, position: 'relative' }}
-      onClick={() => navigate(`${createConcertSlug(concert.title)}`)}
+      onClick={() => navigate(`${createConcertSlug(title)}`)}
     >
       <CardActionArea>
-        <CardMedia component="img" height="140" image={concert.image} />
+        <CardMedia component="img" height="140" image={coverImage} />
         <CardContent>
           <Typography gutterBottom variant="h5" component="div">
-            {concert.title}
+            {title}
           </Typography>
           <Typography variant="body1" color="text.primary">
-            {concert.venue}
+            {venue}
           </Typography>
           <Typography gutterBottom variant="body2" color="text.secondary">
-            {concert.time}
+            {new Date(time.toDate()).toDateString()}
```

Comment:
Don't know the exact type of this time.toDate().. but I think we don't need to convert this in to new Date() again.. we should be able to directly convert this toDateString()..like time.toDate().toDateString() .. unless we need something specific in ToDateString().

Reply:
Done